app/controllers/families_controller.rb

```diff
@@ -1,6 +1,6 @@
 class FamiliesController < ApplicationController
   def index
-    @families = Family.all
+    @families = Family.order(:last_name).page(params[:page]).per(10)
```

Comment:
I would remove the `.order` scope into the model, so it will read:
`Family.by_last_name.page(params[:page]).per(10)`

Especially because you can then reuse that scope in the other family controllers :)

Reply:
Done in sha `a0fa731`